### PR TITLE
Hello Microverse 

### PR DESCRIPTION
**Pull request by AliShahSafdari**
add License into the readme.md file
add test.md file into .gitignore file
change the branch name

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This file must be ignored
```
Comment (ABDULALI3468) [inline .gitignore:1]: **3-** It's a project requirement to add a `test.md file` in the `.gitignore file` as follows but currently it's not having any test.md file.

Consider making a file named `test.md`, add it in .gitignore file, and add the following text in it:- 

```
This file should be ignored by git
```


image

File: README.md [hunk 2]
```diff
@@ -1 +1,150 @@
...
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
```
Comment (ABDULALI3468) [inline README.md:34]: Please consider removing this section. If you want to give credits to Microverse the best way to do it is in the Acknowledgments section.

File: README.md [hunk 3]
```diff
@@ -1 +1,150 @@
...
+
+> Write a message to encourage readers to support your project
+
+If you like this project reach me on my contact
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank in advance
```
Comment (ABDULALI3468) [inline README.md:150]: Please consider either tagging them or naming the ones you are acknowledging to complete the requirements of this section. Currently, you can acknowledge Microverse.

File: README.md [hunk 4]
```diff
@@ -1 +1,150 @@
...
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+> Write a message to encourage readers to support your project
+
+If you like this project reach me on my contact
```
Comment (ABDULALI3468) [inline README.md:144]: First, consider removing this comment on line 142 as its related only to your convenience it has nothing to do with the user to make it more professional. Moreover, add the relative heading of this section which is.

```
Show your support
```

File: README.md [hunk 5]
```diff
@@ -1 +1,150 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (ABDULALI3468) [inline README.md:138]: Please consider adding the relative heading of this section to make it more professional to increase user experience which is.

```
Contributing
```

image

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @AliShahSafdari, 



Indeed you did an exceptional but still, there is room for changes to complete this project. 💯


## Highlights 
✅All linters are checked.
✅The right GitHub flow method has been used.


 ## Required Changes ♻️

**1-** Please consider always using the name of your feature branch as a way to highlight what you did in it (e.g. 'created_sidebar' or 'added_comments_section')  In this case, your branch name is `hello` which is not professional.

> Consider checking out from your current branch to a new branch and for this case, you can name it `feature-1/setup-basic-repo-configurations` or something else that describes what you did in the branch. Remember in the new PR which you will open for another review add the link of this PR in that PR's Description.


**2-** Please consider adding a professional title and description of your PR which describes what you have done in the Pull Request (PR). In this case, it's not describing it clearly.

>   pull_request_desc

**3-** Please consider adding a LICENSE section to our README to make it like the Microverse README template and to meet project requirements. 

image


**4-** As it's not possible to update your commit messages but I request that next time please add a commit message in an imperative way and in the present tense.  

here are some examples of descriptive commit messages written in the imperative tense:

- Add a new feature to the dashboard
- Update error handling for the login form
- Refactor database connection code
- Remove deprecated method from API
- Fix the issue with the navigation menu not displaying properly

> Note that these commit messages clearly state what changes were made and are written in the present tense, which is the recommended style for Git commit messages. Using the imperative tense ("Add", "Update", "Refactor", etc.) helps to convey that the commit is making a specific change, rather than just describing a state.



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AliShahSafdari** [comment]
reopen pull request

File: .gitignore (hunk 1 above)
Comment (AbeeraTahir) [reply .gitignore:1]: - [ ] Hi @AliShahSafdari, as stated by the previous reviewer, please first make a `test.md` file with the text `This file should be ignored by git`, and then add write this file name (`test.md`) into your `.gitignore` file, as also stated in the project requirements:

> A test.md file with the text "This file should be ignored by git".

File: README.md (hunk 2 above)
Comment (AbeeraTahir) [reply README.md:34]: - [ ] As suggested by the previous reviewer, please remove the logo of Microverse from your README file. Optionally, you can also replace it with your own logo, if you have one. 😊

File: README.md (hunk 5 above)
Comment (AbeeraTahir) [reply README.md:138]: - [ ] As suggested by the previous reviewer, please add the `Contributing` heading to this section to make it more professional and to give a better user experience. 🙌

File: README.md (hunk 4 above)
Comment (AbeeraTahir) [reply README.md:144]: - [ ] As suggested by the previous reviewer, please first remove the comment from line 142 as it has nothing to do with the user to make it more professional, and add the relevant heading `Show your support` to this section.

File: README.md (hunk 3 above)
Comment (AbeeraTahir) [reply README.md:150]: - [ ] Please follow this suggestion as well. You can write something like that:

> I would like to thank the Microverse community.

**Review by AbeeraTahir — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

Hi @AliShahSafdari , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✔️ Linters are passing
✔️ Correct GitHub flow is used

 ## Required Changes ♻️
- As suggested by the previous reviewer:

  > Please consider always using the name of your feature branch as a way to highlight what you did in it (e.g. 'created_sidebar' or 'added_comments_section') In this case, your branch name is hello which is not professional.

  You can take help from the suggestion provided by the previous reviewer, or you can also check [this](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch) article, to seek guidance about how to rename a branch.
-  Great job on adding a professional PR title.🙌 But I believe, using numbering or bullet points to highlight your introduced changes in your PR will make it more professional. 😀
- As suggested by the previous reviewer, kindly add a `LICENSE` section to your README file, to make it like the Microverse README template and to meet project requirements.
- _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**